tests/pds/common.go

```diff
@@ -55,6 +55,7 @@ var (
 	replicas                                int32
 	err                                     error
 	supportedDataServices                   []string
+	deps                                    []*pds.ModelsDeployment
```

Comment:
Why are we maintaining global var to track test-specific list of data services, can we define it in the test itself?

Reply:
makes sense, I have made it local to the testcase